src/nyx/environment.cpp

```diff
@@ -826,6 +830,13 @@ bool Environment::parse_cmdline(int argc, char **argv)
 		rawFeatures = FEA_NICK_ALL;
 	}
 
+	//==== Parse optional result file name
+	if (nyxus_result_fname == "")
```

Comment:
is this going to be a mandatory argument?

Reply:
No, optional. Default ('NyxusFeatures' .csv, .arrow, etc) remains the same